CMakeLists.txt

```diff
@@ -455,7 +455,14 @@ if(COMPILER_WARNINGS)
   target_compile_options(kvsWebrtcSignalingClient PUBLIC -Wall -Werror -pedantic -Wextra -Wno-unknown-warning-option)
 endif()
 
-install(TARGETS kvsWebrtcClient kvsWebrtcSignalingClient kvsWebRtcThreadpool
+if(ENABLE_KVS_THREADPOOL)
+  install(TARGETS kvsWebrtcClient kvsWebrtcSignalingClient kvsWebRtcThreadpool
+          LIBRARY DESTINATION lib
+          ARCHIVE DESTINATION lib
+  )
+endif()
+
+install(TARGETS kvsWebrtcClient kvsWebrtcSignalingClient
```

Comment:
Should this be in an else() so that we don't install kvsWebrtcClient kvsWebrtcSignalingClient twice?

Reply:
Fixed. CP error.